benchmark/attri_util.py

```diff
@@ -0,0 +1,132 @@
+import operator
+from dataclasses import dataclass
+from enum import Enum
+from functools import reduce
+from typing import Any, List, Optional, Tuple
+
+
+class ReadOnly:
+    def __init__(self, value):
+        self._value = value
+
+    @property
+    def value(self):
+        return self._value
+
+
+BLAS_OPS = ReadOnly(["addmm", "mv", "addmm", "mm", "outer"])
+
+DEFAULT_WARMUP_COUNT = 100
```

Comment:
My experience is we need a much longer warmup to warrant a stable perf result. I suggest flip the warmup and repeat values.

Reply:
ok.